src/nodes/toJS.ts

```diff
@@ -10,6 +14,8 @@ export interface AnchorData {
 
 export interface ToJSContext {
   anchors: Map<Node, AnchorData>
+  /** Cached anchor and allias nodes in the order they occurred in the document */
+  aliasResolveCache?: (Alias | Scalar | YAMLMap | YAMLSeq)[]
```

Comment:
The typing gets restricted from `Node` by the `node.anchor` check, so this doesn't need to list all the sub-types.

```suggestion
  /** Cached anchor and alias nodes in the order they occur in the document */
  aliasResolveCache?: Node[]
```

Reply:
Done

Reply:
Note the typo fixes in the comment, those still remain.

Reply:
Oh, my bad, fixed